snippets/curl/README.md

```diff
@@ -0,0 +1,19 @@
+### NetBackup API Code Samples for curl
+
+This directory contains code samples to invoke NetBackup REST APIs using PowerShell.
```

Comment:
It should say Curl instead of powershell

Reply:
oops...Thanks for catching this!

Reply:
done 
removed snippets/python/api_requests.pyc from commit too.